dracut/30ignition/coreos-teardown-initramfs.sh

```diff
@@ -100,6 +100,23 @@ propagate_initramfs_hostname() {
     fi
 }
 
+# Persist automatic multipath configuration, if any.
+# When booting with `rd.multipath=default`, the default multipath
+# configuration is written. We need to ensure that the mutlipath configuration
+# is persisted to the final target.
+persist_automatic_multipath() {
+    if [ ! -f /sysroot/etc/multipath.conf ] && [ -f /etc/multipath.conf ]; then
+        echo "info: propagating automatic multipath configuration"
+        cp -v /etc/multipath.conf /sysroot/etc/
+        mkdir -p /sysroot/etc/multipath/multipath.conf.d
+        selinux_relabel /etc/multipath.conf
+        selinux_relabel /etc/multipath
```

Comment:
this will work recursively, right? Is the `selinux_relabel /etc/multipath/multipath.conf.d` required?

Reply:
I'm not sure about `selinux_relabel` being being recursive. But `.d` is required.